packages/dashboard/src/customization/widgets/types.ts

```diff
@@ -72,7 +72,6 @@ export type LineAndScatterStyles = {
   line?: LineStyles;
   aggregationType?: AggregateType;
   resolution?: string;
-  visible?: boolean; // defaults to true
```

Comment:
this is a public API change

Reply:
We discussed this change when this ticket was created. Since it is optional, we decided to remove it completely.

Reply:
https://quip-amazon.com/KgFkAMKfbT8n/PFR-27-HideShow-property-Feature-requirement